pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -256,12 +218,12 @@ func GetRegistries(ctx *types.SystemContext) ([]Registry, error) {
 	}
 	if len(v1Registries) > 0 {
 		if len(registries) > 0 {
-			return nil, fmt.Errorf("mixing sysregistry v1/v2 is not supported")
+			return nil, errors.Wrap(&InvalidRegistries{}, "mixing sysregistry v1/v2 is not supported")
```

Comment:
Shouldn’t this, and all other uses of `InvalidRegistries`, be `return nil, InvalidRegistries{s:"…"}` ?  It’s surprising for no user of `InvalidRegistries` to set the `s` field (why does it exist at all, then?), and if I’m reading it correctly, the `errors.Wrapf` use would result in an extra trailing `": "`.

Reply:
(It might also be more convenient to use `type InvalidRegistries string`, because the error case would be just `return nil, InvalidRegistries("…")`.  OTOH it would be easier to accidentally mix the types than when a `struct` is involved.  Either is perfectly fine.)

Reply:
Good catch, changed to `&InvalidRegistries{s: ...}`.